Makefile

```diff
@@ -1,5 +1,7 @@
 VERSION=v0.2.0
```

Comment:
Lets add a `VERSION` file to read this variable from that file, I am fine to also do it in a follow up.

Reply:
I addressed your comment, what do you think about it? @lilic